### ISSUE TITLE
AssertationError in `calculate_imbalance_fees`

### ISSUE THREAD

**Issue by palango** [raiden-network/raiden#4446]
Found in #4346

Probably related to #4436 

```
Traceback (most recent call last):
  File "/home/manuel/.virtualenvs/webui-verify/bin/raiden", line 11, in <module>
    load_entry_point('raiden', 'console_scripts', 'raiden')()
  File "/home/manuel/Projects/raiden/raiden/raiden/__main__.py", line 13, in main
    run(auto_envvar_prefix="RAIDEN")  # pylint: disable=no-value-for-parameter
  File "/home/manuel/.virtualenvs/webui-verify/lib/python3.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/home/manuel/.virtualenvs/webui-verify/lib/python3.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/home/manuel/.virtualenvs/webui-verify/lib/python3.7/site-packages/click/core.py", line 1114, in invoke
    return Command.invoke(self, ctx)
  File "/home/manuel/.virtualenvs/webui-verify/lib/python3.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/manuel/.virtualenvs/webui-verify/lib/python3.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/home/manuel/.virtualenvs/webui-verify/lib/python3.7/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/home/manuel/Projects/raiden/raiden/raiden/ui/cli.py", line 507, in run
    app = runner.run()
  File "/home/manuel/Projects/raiden/raiden/raiden/ui/runners.py", line 244, in run
    return self._start_services()
  File "/home/manuel/Projects/raiden/raiden/raiden/ui/runners.py", line 95, in _start_services
    app_ = run_app(**self._options)
  File "/home/manuel/Projects/raiden/raiden/raiden/ui/app.py", line 282, in run_app
    raiden_app.start()
  File "/home/manuel/Projects/raiden/raiden/raiden/app.py", line 125, in start
    self.raiden.start()
  File "/home/manuel/Projects/raiden/raiden/raiden/raiden_service.py", line 432, in start
    self._initialize_channel_fees(self.config.get("use_imbalance_penalty", False))
  File "/home/manuel/Projects/raiden/raiden/raiden/raiden_service.py", line 983, in _initialize_channel_fees
    self.handle_and_track_state_changes([state_change])
  File "/home/manuel/Projects/raiden/raiden/raiden/raiden_service.py", line 569, in handle_and_track_state_changes
    for greenlet in self.handle_state_changes(state_changes):
  File "/home/manuel/Projects/raiden/raiden/raiden/raiden_service.py", line 589, in handle_state_changes
    new_state, raiden_event_list = self.wal.log_and_dispatch(state_changes)
  File "/home/manuel/Projects/raiden/raiden/raiden/storage/wal.py", line 87, in log_and_dispatch
    latest_state, all_events = self.state_manager.dispatch(state_changes)
  File "/home/manuel/Projects/raiden/raiden/raiden/transfer/architecture.py", line 261, in dispatch
    iteration = self.state_transition(next_state, state_change)
  File "/home/manuel/Projects/raiden/raiden/raiden/transfer/node.py", line 1143, in state_transition
    iteration = handle_state_change(chain_state, state_change)
  File "/home/manuel/Projects/raiden/raiden/raiden/transfer/node.py", line 795, in handle_state_change
    state_change=state_change,
  File "/home/manuel/Projects/raiden/raiden/raiden/transfer/node.py", line 156, in subdispatch_by_canonical_id
    pseudo_random_generator=chain_state.pseudo_random_generator,
  File "/home/manuel/Projects/raiden/raiden/raiden/transfer/token_network.py", line 374, in state_transition
    pseudo_random_generator=pseudo_random_generator,
  File "/home/manuel/Projects/raiden/raiden/raiden/transfer/token_network.py", line 61, in subdispatch_to_channel_by_id
    pseudo_random_generator=pseudo_random_generator,
  File "/home/manuel/Projects/raiden/raiden/raiden/transfer/channel.py", line 2508, in state_transition
    iteration = handle_action_update_fee(channel_state=channel_state, update_fee=state_change)
  File "/home/manuel/Projects/raiden/raiden/raiden/transfer/channel.py", line 1939, in handle_action_update_fee
    channel_capacity=get_capacity(channel_state)
  File "/home/manuel/Projects/raiden/raiden/raiden/transfer/mediated_transfer/mediation_fee.py", line 110, in calculate_imbalance_fees
    x_values = linspace(TokenAmount(0), channel_capacity, num_base_points)
  File "/home/manuel/Projects/raiden/raiden/raiden/transfer/mediated_transfer/mediation_fee.py", line 84, in linspace
    assert num > 1
AssertionError
```

**Comment by palango** [comment]
@manuelwedler Which version did that happen on? Seems like the code changed in the meantime, can you reproduce this on latest develop?

**Comment by manuelwedler** [comment]
This happened on `fd5b2770d`, where I just changed the webui version.
I can't reproduce the error on the latest develop.
But I had the same error with the released v0.100.4 binary before.
